### DISCUSSION TITLE
Instalation error

### DISCUSSION THREAD

**Discussion by waally1993** [talmolab/sleap#1733] [Help!] [answered]
I keep getting this error:

(base) C:\Users\wally>mamba create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3
usage: mamba [-h] [--version] [--slow SLOW] [--enable-coverage] [--coverage-file COVERAGE_FILE] [--format FORMAT]
             [--no-color] [--tags TAGS]
             [specs ...]
mamba: error: unrecognized arguments: -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3

Thank you!

**Comment by eberrigan** [comment]
Hi!

- What version of `mamba` are you using and what operating system do you have?
-  Do other `mamba` commands work in your shell?
-  You could try splitting up:
- 1. Make the environment: `mamba create -n sleap python=3.7`
- 2. Activate the environment: `mamba activate sleap`
- 3. Install `mamba install -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3`
- Please let us know where you see errors.

Thanks!

-Elizabeth

**Comment by waally1993** [comment]
Hi, thanks for the response.

I have mamba version 0.11.3. 
My operating systems is 64-bit operating system, x64-based processor.
Mamba seems to be working as I write mamba -h and I received info. 

But the error persists>

(base) C:\Users\wally>mamba create -n sleap
usage: mamba [-h] [--version] [--slow SLOW] [--enable-coverage] [--coverage-file COVERAGE_FILE] [--format FORMAT] [--no-color]
             [--tags TAGS]
             [specs ...]
mamba: error: unrecognized arguments: -n sleap

I am installing mamba as Sleap installation web recommends: 

conda update -n base conda
conda install -n base conda-libmamba-solver
conda config --set solver libmamba

**Reply by eberrigan** [reply, marked as answer]
I see. It appears you've set the solver to `libmamba`, but it seems you haven't installed `mamba`. It's a bit surprising that any `mamba` commands are working for you at all. Did you also install `mamba` following the directions provided here: [Miniforge](https://github.com/conda-forge/miniforge#mambaforge)? With Windows, it typically comes as an installer.

Could you confirm if you have Miniforge installed on your computer? If you do, I would recommend using that. Please let us know if you need further assistance.

**Reply by waally1993** [reply]
I did re install mamba following your guidance. Now I have a problem with this step>

mamba install -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3

Looking for: ['sleap=1.3.3']

conda-forge/win-64                                          Using cache
conda-forge/noarch                                          Using cache
nvidia/win-64 (check zst)                           Checked  0.4s
nvidia/noarch (check zst)                           Checked  0.2s
sleap/win-64 (check zst)                            Checked  0.9s
sleap/noarch (check zst)                            Checked  0.7s
anaconda/win-64 (check zst)                         Checked  0.1s
anaconda/noarch (check zst)                         Checked  0.2s
nvidia/win-64                                      128.0kB @   1.3MB/s  0.1s
nvidia/noarch                                        5.2kB @  28.9kB/s  0.2s
pkgs/main/noarch                                              No change
pkgs/main/win-64                                              No change
pkgs/r/noarch                                                 No change
pkgs/r/win-64                                                 No change
pkgs/msys2/noarch                                             No change
pkgs/msys2/win-64                                             No change
anaconda/noarch                                    462.3kB @ 784.4kB/s  0.3s
sleap/win-64                                         2.4kB @   3.8kB/s  0.6s
sleap/noarch                                       116.0 B @ 181.0 B/s  0.6s
anaconda/win-64                                      2.8MB @   3.3MB/s  0.9s

Pinned packages:
  - python 3.7.*


warning  libmamba Problem type not implemented SOLVER_RULE_STRICT_REPO_PRIORITY
Could not solve for environment specs
The following package could not be installed
└─ sleap 1.3.3**  is not installable because it requires
   └─ cuda-nvcc 11.3.* , which conflicts with any installable versions previously reported.

I dont have Nvidia in this PC.

**Reply by waally1993** [reply]
I was able to install it using this comand> pip install sleap[pypi]

Thank you very much!
